### DISCUSSION TITLE
translation plugin

### DISCUSSION THREAD

**Discussion by dlaflamme1** [liedekef/events-made-easy#241] [General]
Hello, I'm trying to translate the description of my events with the translatePress extension but that didnt translate. Witch plugin I can use to translate descriptions of my events on my website? Its the only things I didnt find how I can do it. And the administrator of the translatePress plugins said he dont know if i can use his plugin to translate the events of your plugin
https://www.ginger-restaurant.com/events/big-city-6/

**Comment by liedekef** [comment]
Next to translation of the plugin itself (mostly backend), see also https://www.e-dynamics.be/wordpress/category/documentation/22-language-support/

**Comment by dlaflamme1** [comment]
thank you I only have one last problem
I dont know how to make a traductioon of the button En savoir plus and how to change the hour like 18:30 to 6:30pm.
https://www.ginger-restaurant.com/en/prestations/
![Capture d’écran 2023-07-20 141113](https://github.com/liedekef/events-made-easy/assets/127334032/e9e8f2d4-69e3-4383-8740-4741f90013f1)
thank you in advance

**Reply by liedekef** [reply]
EME can set the date in any format you want, see the doc on that: https://www.e-dynamics.be/wordpress/category/documentation/7-placeholders/7-2-events/ at the bottom
Concerning the "En savoir plus": it seems that is your own text inside the event link (so probably using something like #_PAGEURL somewhere in your format, so check that ...)
